src/xcode/ENA/ENA/Source/Models/Task Scheduling/ENATaskScheduler.swift

```diff
@@ -22,115 +22,72 @@ import UIKit
 enum ENATaskIdentifier: String, CaseIterable {
 	// only one task identifier is allowed have the .exposure-notification suffix
 	case exposureNotification = "exposure-notification"
```

Comment:
do we need the value: "exposure-notification" in the enum?

Reply:
it is required, we assemble the task identifier from this enum. the special handling of the task is provided by apple based on this exact name.

Reply:
Hi @johannesrohwer , 
I got confused here. The code looks like:
```swift
enum ENATaskIdentifier: String, CaseIterable {
	// only one task identifier is allowed have the .exposure-notification suffix
	case exposureNotification = "exposure-notification"

	var backgroundTaskSchedulerIdentifier: String {
		guard let bundleID = Bundle.main.bundleIdentifier else { return "invalid-task-id!" }
		return "\(bundleID).\(rawValue)"
	}
}
```
The bundle id is $(PRODUCT_BUNDLE_IDENTIFIER).exposure-notification aka, but we appended another suffix, "exposure-notification", I think it is duplicated. What do you think so?

Reply:
ah. my bad. Sorry, just ignore it. I got confused with **Permitted background task** and **bundle identifier**